### ISSUE TITLE
What does "unsupervised STS" mean?

### ISSUE THREAD

**Issue by null-id** [UKPLab/sentence-transformers#428]
How could SBERT (before fine-tuning) work without supervision? I don't quite understand the content in the paper section 4.

I mean, is it fair to compare BERT without fine-tuning and SBERT after fine-tuning with labeled data?

**Comment by nreimers** [comment]
"Unsupervised STS" means without any explicit training data for STS. But other training data can be used.

"Supervised STS" means with training data for STS.

If you have training data for STS, the results for STS are of course better than having not explicit training data.

**Comment by null-id** [comment]
Yeah, sorry I know where I am wrong. The triplet loss is actually can fit unsupervised training, or more precisely, self-supervised training. Is it right?
